tests/lib.rs

```diff
@@ -280,3 +281,91 @@ fn block_height() {
             .unwrap()
     )
 }
+
+#[test]
+fn proof_verifier() {
+    use dusk_plonk::prelude::*;
+    use transfer_circuits::ExecuteCircuit;
+
+    let contract = ProofVerifier::new();
+
+    let store = MS::new();
+
+    let code =
+        include_bytes!("contracts/proof_verification/proof_verification.wasm");
+
+    let contract = Contract::new(contract, code.to_vec(), &store).unwrap();
+
+    let mut network = NetworkState::<StandardABI<_>, MS>::default();
+
+    let contract_id = network.deploy(contract).unwrap();
+
+    let mut gas = GasMeter::with_limit(1_000_000_000);
+
+    let old_crs = rusk_profile::get_common_reference_string();
+    let pp = unsafe {
+        let buff = std::fs::read("tests/pub_params_dev.bin")
+            .expect("Error reading from PubParams file");
```

Comment:
This is not unsafe it shouldn't be there, but the API will change shortly anyway so no need to change anything here.